srcpkgs/font-adobe-source-code-pro/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'font-adobe-source-code-pro'
+pkgname=font-adobe-source-code-pro
+version=1.010
+revision=2
+noarch=yes
+build_style=fetch
+depends="font-util"
+short_desc="Monospaced font family for user interface and coding environments"
+homepage="https://adobe-fonts.github.io/source-code-pro/"
+maintainer="Piero La Terza <platerza@protonmail.com>"
+license="OFL-1.1"
+distfiles="https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Roman.otf
+	https://github.com/adobe-fonts/source-code-pro/releases/download/variable-fonts/SourceCodeVariable-Italic.otf"
```

Comment:
prefix the new line with a whitespace like it is done with dependencies, see gtk+3